common-lib/src/cmr/common/api/context.clj

```diff
@@ -61,6 +61,6 @@
   ([context]
    (context->user-id context "Valid user token required."))
   ([context msg]
-   (if-let [token (:token context)]
+   (if-let [_token (:token context)]
```

Comment:
I changed this function - in order not to get a merge conflict you should change this to
(if (:token context) 

you don't need the let statement because the token variable is not used.

Reply:
will show up in my next update.